plotly/offline/offline.py

```diff
@@ -56,20 +58,40 @@ def init_notebook_mode():
         raise ImportError('`iplot` can only run inside an IPython Notebook.')
 
     global __PLOTLY_OFFLINE_INITIALIZED
-    # Inject plotly.js into the output cell
-    script_inject = (
-        ''
-        '<script type=\'text/javascript\'>'
-        'if(!window.Plotly){{'
-        'define(\'plotly\', function(require, exports, module) {{'
-        '{script}'
-        '}});'
-        'require([\'plotly\'], function(Plotly) {{'
-        'window.Plotly = Plotly;'
-        '}});'
-        '}}'
-        '</script>'
-        '').format(script=get_plotlyjs())
+    global __PLOTLY_USE_CDN
+
+    __PLOTLY_USE_CDN = connected
+
+    if connected:
+        # Inject plotly.js into the output cell
+        script_inject = (
+            ''
+            '<script>'
+            # 'if(!window.Plotly) {{'
```

Comment:
There were some persistence and notebook sharing issues from time to time when I left them in. So I just took them out for now.